src/RabbitMQAttribute.cs

```diff
@@ -34,6 +34,14 @@ public sealed class RabbitMQAttribute : Attribute
         [AppSetting]
         public string Password { get; set; }
 
+        /// <summary>
+        /// Gets or sets the vhost to use when connecting to RabbitMQ using HostName / UserName / Password
+        /// Optional
+        /// <see href="https://www.rabbitmq.com/vhosts.html"/>
+        /// </summary>
+        [AppSetting]

```

Comment:
@brettsam Do you recommend `[AppSetting]` or `[AutoResolve]` here? In what cases should one be used over the other ?